.gitpod/start_repo.sh

```diff
@@ -26,7 +27,7 @@ if [ -d ${GITPOD_REPO_ROOT}/${reponame} ]; then
   # Now that composer install has been done, if we were using an empty
   # .ddev/config.yaml, we'll do a real ddev config
   if [ ! -s .ddev/config.yaml ]; then
-    ddev config --auto
+    ddev config --auto --project-name="${clean_reponame}"
```

Comment:
```suggestion
    ddev config --project-name="${clean_reponame}"
```

Reply:
If you don't mind that and it tests out for you, you can just commit it and add it, by clicking the "Commit suggestion"